vector-stores/spring-ai-gemfire/src/main/java/org/springframework/ai/vectorstore/GemFireVectorStore.java

```diff
@@ -378,9 +446,12 @@ public List<Document> similaritySearch(SearchRequest request) {
 		}
 	}
 
-	public void createIndex() throws JsonProcessingException {
+	public void createIndex() {
 		try {
-			CreateRequest createRequest = new CreateRequest(INDEX_NAME);
+			CreateRequest createRequest = new CreateRequest();
+			createRequest.setName(INDEX_NAME);
+			createRequest.setBeamWidth(20);
```

Comment:
Why are we setting the beamwidth to 20?
Shouldn't these values use the confgured values?